packages/fiori/src/SettingItem.ts

```diff
@@ -0,0 +1,278 @@
+ *
+ * <b>Note:</b> Disabled and read-only states shouldn't be used together.
+ *
+ * @constructor
+ * @extends UI5Element
+ * @experimental
+ * @public
+ */
+class SettingItem extends UI5Element {
+	eventDetails!: {
+		"_collapse": void,
+		"view-select": SettingItemViewSelectEventDetail,
+		"back-click": SettingItemBackClickEventDetail,
+
+	}
+	/**
+	 * Defines the text of the item.
+	 *
+	 *
+	 * @public
+	 * @default undefined
+	 */
+	@property({ type: String })
+	text = "";
+
+	/**
+	 * Defines the tooltip of the component.
+	 *
+	 * A tooltip attribute should be provided, in order to represent meaning/function, when the component is collapsed(icon only is visualized).
+	 * @default undefined
+	 * @public
+	 */
+	@property({ type: String })
+	tooltip = "";
+
+	/**
+	 * Defines the headerTitle of the item.
+	 *
+	 * @public
+	 * @default undefined
+	 */
+	@property({ type: String })
+	headerTitle?: string;
+
+	/**
+	 * Shows item tab.
+	 *
+	 * @public
+	 * @default false
+	 */
+	@property({ type: Boolean })
+	selected = false;
+
+	/**
+	 * Defines whether the component is in disabled state.
+	 *
+	 * **Note:** A disabled component is completely noninteractive.
+	 * @default false
+	 * @public
+	 */
+	@property({ type: Boolean })
+	disabled = false;
+
+	/**
+	 * Indicates whether a loading indicator should be shown.
+	 * @default false
+	 * @public
+	 */
+	@property({ type: Boolean })
+	loading = false;
+
+	/**
+	 * Defines the icon of the component.
+	 *
+	 * @default undefined
+	 * @public
+	 */
+	@property({ type: String })
+	icon = "globe";
+
+	/**
+	 * Defines the accessible ARIA name of the component.
+	 * @default undefined
+	 * @public
+	 */
+	@property()
+	accessibleName?: string;
+
+	/**
+	 * Defines the tab views of the setting item.
+	 *
+	 * The tab views are displayed by default if there is no selected page view.
+	 * @public
+	 */
+	@slot({
+		type: HTMLElement,
+		"default": true,
+		individualSlots: true,
+		invalidateOnChildChange: {
+			properties: true,
+			slots: false,
+		},
+	})
+	tabs!: Array<SettingView>;
+
+	/**
+	 * Defines the page views of the setting item.
+	 *
+	 * If there are no tab views, the first page view will be shown unless there is selected one. If there is selected page view it will be shown no matter if there are tab views.
+	 *
+	 * @public
+	 */
+	@slot({
+		type: HTMLElement,
+		individualSlots: true,
+		invalidateOnChildChange: {
+			properties: true,
+			slots: false,
+		},
+	})
+	pages!: Array<SettingView>;
+
+	/**
+	 * @private
+	 */
+	_individualSlot?: string;
+
+	get _hasSelectedPageView() {
+		return this.pages.some(view => view.selected);
+	}
+
+	get _selectedPageView() {
+		return this.pages.find(view => view.selected) || this.pages[0];
+	}
+
+	get ariaLabelledByText() {
+		return `${this.text} ${this.accessibleName}`.trim();
+	}
+
+	get _tooltip() {
+		return this.tooltip ? this.tooltip : this.text;
+	}
+
+	get _icon() {
+		return `sap-icon://${this.icon}`;
```

Comment:
do we need `sap-icon` prefix? icons from the SAP-icons fonts can be used in following name.

sap-icon://home => home
sap-icon://BusinessSuiteInAppSymbols/icon-2x2-grid-layout => business-suite/icon-2x2-grid-layout
sap-icon://SAP-icons-TNT/aggregation => tnt/aggregation